packages/core/src/logger/defaultLogger.ts

```diff
@@ -0,0 +1,8 @@
+import { Level, LoggerInterface } from '@apimatic/core-interfaces';
+
+export class ConsoleLogger implements LoggerInterface {
+  public log(level: Level, message: string, params: Record<string, any>): void {
+    // tslint:disable-next-line:no-console
+    console.log('level : ' + level + ', message : ' + message, params);
```

Comment:
Please show me an example of what the log lines will look like.

When logging into the console, we do not need to explicitly spell out "level : " or "message : ". Most loggers will format their console log output like this:
```suggestion
    console.log(level + ': ' + message, params);
```

Reply:
This was changed as suggested